index.html

```diff
@@ -0,0 +1,115 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mobile First Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header">
+        <div class="placeholder">
+            <img src="images/Logo Placeholder.png" alt="My Logo">
+        </div>
+        
+        <div class="hamburger">
+            <a href="#">
+                <img src="images/hamburger.png" alt="Hamburger Icon">
+            </a>
+        </div>
```

Comment:
- [ ] Great work implementing the hamburger button :+1:, but enclosing the element within a `nav` tag is necessary for better structure and semantics. By wrapping the button inside a `nav` tag, it signifies that it's part of the website's navigation and enhances the accessibility and functionality of the website. Additionally, it makes it easier for search engines to understand the website's layout and improves its SEO :+1:.